connector/connect/client/jvm/src/test/resources/log4j2.properties

```diff
@@ -32,7 +32,7 @@ appender.console.type = Console
 appender.console.name = console
 appender.console.target = SYSTEM_ERR
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %t: %m%n%ex
+appender.console.layout.pattern = %d{HH:mm:ss.SSS} %p %c: %maxLen{%m}{512}%n%ex{8}%n
```

Comment:
Did we mention this log style change in the PR description clearly? It's more than a simple truncation.

Reply:
updated in the section of PR desc, thanks @dongjoon-hyun